app/usecases/users.py

```diff
@@ -314,7 +315,7 @@ async def delete_one_by_user_id(user_id: int, /) -> None | Error:
         # TODO: split this to make it more clear what's being done
         #       at the usecase layer
         await users.anonymize_one_by_user_id(user_id)
-
+        await assets.delete_avatar_by_user_id(user_id)
```

Comment:
more clearly separate from following concern
```suggestion
        await assets.delete_avatar_by_user_id(user_id)

```